tests/language-feature/function-calls/forceinline-basic-block.slang

```diff
@@ -0,0 +1,38 @@
+//TEST:SIMPLE(filecheck=CHECK): -stage compute -entry computeMain -target spirv -O0 -g3
+RWStructuredBuffer<int> outputBuffer;
+
+[ForceInline]
+int inlineSingleBasicBlock1(int value1, int value2)
+{
+    // Simple operation that should be inlined
+    return value1 * 2 + value2;
+}
+
+[ForceInline]
+int inlineSingleBasicBlock2(int value1, int value2)
+{
+    int result1 = inlineSingleBasicBlock1(10, 20);
+    // Simple operation that should be inlined
+    return value1 * 2 + value2 + result1;
```

Comment:
Have you tested this with nsight? I don't think the current PR handles `debugNoScope` correctly, and the call stack info for line 16 may not be displayed correctly as a result.